velox/serializers/tests/PrestoSerializerTest.cpp

```diff
@@ -1090,6 +1091,23 @@ TEST_P(PrestoSerializerTest, longDecimal) {
   testRoundTrip(vector);
 }
 
+TEST_P(PrestoSerializerTest, uuid) {
+  std::vector<int128_t> uuidValues(200);
+
+  for (int row = 0; row < uuidValues.size(); row++) {
+    uuidValues[row] = (int128_t) 0xD1 << row % 120;
+  }
+  auto vector = makeFlatVector<int128_t>(uuidValues, UUID());
```

Comment:
Use variation https://github.com/facebookincubator/velox/blob/main/velox/vector/tests/utils/VectorTestBase.h#L183 of makeFlatVector to avoid the for loops above.